src/cets.erl

```diff
@@ -446,11 +456,11 @@ ets_delete_objects(_Tab, []) ->
     ok.
 
 has_remote_pid(RemotePid, Servers) ->
-    lists:member(RemotePid, Servers).
+    lists:keymember(RemotePid, 1, Servers).
 
 reply_updated({Mon, Pid}) ->
-    %% We really don't wanna block this process
-    erlang:send(Pid, {updated, Mon, self()}, [noconnect, nosuspend]).
+    Pid ! {cets_updated, Mon, self()},
```

Comment:
Why did we drop the `[noconnect, nosuspend]` options here?

Reply:
they both would not sent a message and just return quickly (if detected busy dist port or require connect).
We could use `[noconnect, nosuspend]` but something like:

```erlang
case erlang:send(Pid, M, [noconnect, nosuspend]) of
    ok -> ok;
    Other -> spawn(fun() -> Pid ! M end) % Message is not sent if function returns not ok
end.
```

`noconnect` - pretty useless, we have nodes connected at this point. Especially useless with newer `async_dist=true`. I am not sure about the behaviour when node disappears and we have still messages addressed to that node... could keep `noconnect` probably :D